src/main/java/com/xiaomi/infra/pegasus/rpc/async/ClusterManager.java

```diff
@@ -56,7 +58,12 @@ public ClusterManager(ClientOptions opts) throws IllegalArgumentException {
     replicaGroup = getEventLoopGroupInstance(opts.getAsyncWorkers());
     metaGroup = getEventLoopGroupInstance(1);
     tableGroup = getEventLoopGroupInstance(1);
-    enableAuth = opts.isEnableAuth();
+    if (opts.isEnableAuth()) {
+      ReplicaSessionInterceptor hook =
+          new SecurityReplicaSessionInterceptor(
+              opts.getJaasConf(), opts.getServiceName(), opts.getServiceFQDN());
+      ReplicaSessionInterceptorManager.instance().addHook(hook);
```

Comment:
Just as your previous commet about `TableInterceptor`: https://github.com/XiaoMi/pegasus-java-client/pull/125#discussion_r481045604, the `ReplicaSessionInterceptor` can be put into `ReplicaSessionInterceptorManager`